lib/segment/src/data_types/vectors.rs

```diff
@@ -399,28 +410,14 @@ pub struct NamedSparseVector {
     pub vector: SparseVector,
 }
 
-/// Vector data separator for named and unnamed modes
-/// Unnamed mode:
-///
-/// {
-///   "vector": [1.0, 2.0, 3.0]
-/// }
-///
-/// or named mode:
-///
-/// {
-///   "vector": {
-///     "vector": [1.0, 2.0, 3.0],
-///     "name": "image-embeddings"
-///   }
-/// }
-#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
+#[derive(Debug, Deserialize, Serialize, Clone)]
```

Comment:
Do we need `Deserialize` and `Serialize`?

Reply:
very good catch :+1: 

https://github.com/qdrant/qdrant/pull/4014/commits/ff0b47db1a5c8f2475a1889253fef6d17087fe95